xbmc/pvr/windows/GUIWindowPVRGuide.h

```diff
@@ -43,6 +43,7 @@ namespace PVR
 
   protected:
     void UpdateSelectedItemPath();
+    virtual std::string GetDirectoryPath(void) { return ""; }
```

Comment:
Could you add `override` since the parent is a pure virtual? Same in the other derived class.